README.md

```diff
@@ -6,6 +6,10 @@
 
 - REACT_APP_PUBLIC_INFURA_ID
 
+- REACT_APP_PROVIDER_URL_1 (Ethereum Provider)
+
+- REACT_APP_PROVIDER_URL_137 (Polygon Provider)
```

Comment:
I think we wanted to offer flexibility on defining providers instead of always being forced to use Infura for everything, even if it had an infura option.